Source/XCDBLD/BuildArguments.swift

```diff
@@ -6,10 +6,10 @@ public struct BuildArguments {
 	/// binary.
 	public enum BitcodeGenerationMode: String {
 		/// Only bitcode marker will be embedded.
-		case marker = "marker"
+		case marker
 
 		/// Full bitcode will be embedded.
-		case bitcode = "bitcode"
+		case bitcode
```

Comment:
I prefer the explicitness since the value matches to Xcode settings so that is really important.